docs/tutorial/03_configure_sdcore.md

```diff
@@ -1,109 +1,33 @@
 # 3. Configure SD-Core
 
-Configuring the 5G core network is done making HTTP requests to the `webui` service. Here we will 
-create a subscriber, a device group and a network slice.
+Configuration of the 5G core network is done using the NMS (Network Management System) graphical
+interface. Here we will create a subscriber, a device group and a network slice.
 
-Run `juju status` and note the IP address next to the `webui` unit.
+Access the NMS at `https://<model name>-nms.<your hostname>` and click on the `Configure` button
```

Comment:
This does not feel like it's going to work.  Where is the DNS entry for `<model name>-nms.<your hostname>`, as that is what the web browser will attempt to look up.

Reply:
Of course. I missed that part, because I already had that done. Nice catch.
Now it says you need to put a relevant entry to your `/etc/hosts`.